lib/Driver/ToolChains.cpp

```diff
@@ -512,6 +512,10 @@ ToolChain::constructInvocation(const InterpretJobAction &job,
   Arguments.push_back(context.Args.MakeArgString(context.OI.ModuleName));
 
   context.Args.AddAllArgs(Arguments, options::OPT_l, options::OPT_framework);
+  
+  if (context.Args.hasArg(options::OPT_e)) {
+    context.Args.AddAllArgs(Arguments, options::OPT_e);
```

Comment:
It's okay to just call `AddAllArgs` directly here; if there aren't any, it will add all zero of them. :-)